jobserver/github.py

```diff
@@ -389,6 +456,28 @@ def get_repos_with_dates(self, org):
                 "topics": topics,
             }
 
+    def set_repo_topics(self, org, repo, topics):
+        path_segments = [
+            "repos",
+            org,
+            repo,
+            "topics",
+        ]
+        url = self._url(path_segments)
+
+        payload = {
+            "names": topics,
+        }
+
+        headers = {
+            "Accept": "application/vnd.github.v3+json",
+        }
+        r = self._put(url, headers=headers, json=payload)
```

Comment:
I think this implementation may wipe any existing topics that have been manually added? 

Whilst likely not a problem in the interactive case, I see a foot gun waiting to happen. e.g. we could accidentally remove our manually created `github-releases` topics via this API call.

Reply:
Good point.  I'll get the current topics and merge in anything we pass to avoid the footgun 👍  (You can't do add/remove via the API, you have to do GET/PUT)

Reply:
yep, familiar with this pain :(

Reply:
topics are part of the get_repo response so I can check them there 🎉  Updated the calls in the create_repo command to make sure we use the existing ones and only update if it's not already there.